linked_list/node.py

```diff
@@ -2,15 +2,33 @@ class Node(object):
     """
     Node class representing each of the linked nodes in the list.
     """
-
+    
     def __init__(self, elem, next=None):
-        pass
+        self.elem = elem            # elem part
+        self.next = next            # next node
 
+    # Node(9) > Node(X)
     def __str__(self):
-        pass
+        # return value must be string object
+        if self.next == None:
+            return "Node({}) > /".format(self.elem)
+        else:
+            next_node = self.next
+            return "Node({}) > Node({})".format(self.elem, next_node.elem)
 
+    # x==y calls x.__eq__(y)
     def __eq__(self, other):
-        pass
-
+        if other == None:
```

Comment:
You are right. I should think like this. Since the values were either `int` or `None` I didn't bother.